microservice-vote/pom.xml

```diff
@@ -193,6 +193,12 @@
             <artifactId>demo-bootstrap</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>net.wasdev.wlp.starters.ms-builder</groupId>
```

Comment:
Please use the following snippet 
<dependency>
    <groupId>org.eclipse.microprofile.config</groupId>
    <artifactId>microprofile-config-api</artifactId>
    <version>1.1</version>
</dependency>